xml/schema/logixng/digital-actions/web-request-5.3.5.xsd

```diff
@@ -0,0 +1,86 @@
+<!--                                                                        -->
+<!-- JMRI is distributed in the hope that it will be useful, but WITHOUT    -->
+<!-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or  -->
+<!-- FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License  -->
+<!-- for more details.                                                      -->
+
+<!-- This file contains definitions for LogixNG                             -->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:xsi ="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns:docbook="http://docbook.org/ns/docbook"
+           xmlns:jmri="http://jmri.org/xml/schema/JMRIschema"
+           xsi:schemaLocation="
+                http://jmri.org/xml/schema/JMRIschema http://jmri.org/xml/schema/JMRIschema.xsd
+                http://docbook.org/ns/docbook http://jmri.org/xml/schema/docbook/docbook.xsd
+            "
+        >
+
+    <xs:complexType name="LogixNG_DigitalAction_WebRequestType">
+      <xs:annotation>
+        <xs:documentation>
+          Define the XML stucture for storing the contents of a WebRequest class.
+        </xs:documentation>
+        <xs:appinfo>
+            <jmri:usingclass configurexml="true">jmri.jmrit.logixng.digital.actions.configurexml.WebRequestXml</jmri:usingclass>
+        </xs:appinfo>
+      </xs:annotation>
+        
+            <xs:sequence>
+
+              <xs:element name="systemName" type="systemNameType" minOccurs="1" maxOccurs="1"/>
+              <xs:element name="userName" type="userNameType" minOccurs="0" maxOccurs="1"/>
+              <xs:element name="comment" type="commentType" minOccurs="0" maxOccurs="unbounded"/>
+
+              <xs:element name="Socket" minOccurs="1" maxOccurs="unbounded">
+                <xs:complexType>
+                  <xs:sequence>
+                    <xs:element name="socketName" type="xs:string" minOccurs="1" maxOccurs="1"/>
+                    <xs:element name="systemName" type="systemNameType" minOccurs="0" maxOccurs="1"/>
+                  </xs:sequence>
+                </xs:complexType>
+              </xs:element>
+
+              <xs:element name="requestMethod" type="LogixNG_SelectEnumType" minOccurs="0" maxOccurs="1" />
+              <xs:element name="url" type="LogixNG_SelectStringType" minOccurs="0" maxOccurs="1" />
+              <xs:element name="userAgent" type="LogixNG_SelectStringType" minOccurs="0" maxOccurs="1" />
+
+              <xs:element name="localVariableForPostContent" type="xs:string" minOccurs="0" maxOccurs="1" />
+              <xs:element name="localVariableForResponseCode" type="xs:string" minOccurs="0" maxOccurs="1" />
+              <xs:element name="localVariableForReplyContent" type="xs:string" minOccurs="0" maxOccurs="1" />
+              <xs:element name="_localVariableForCookies" type="xs:string" minOccurs="0" maxOccurs="1" />
+
```

Comment:
Is the leading underscore here intentional?

Reply:
Thanks for catching that! Fixed.